ops/cli.py

```diff
@@ -6,6 +6,7 @@
 
 usage: python3 cli.py -h
 """
```

Comment:
I have zero issue with updating this file as part of this refactoring, but I am gonna need to work on this for GH-877 so I either want to get this merged or I'll have to base my work on this branch